### DISCUSSION TITLE
A better, more accessible/traceable/reliable way to discuss community topics

### DISCUSSION THREAD

**Discussion by spbrogan** [tianocore/edk2#2595] [General]
After talking with @mdkinney and a few others about trying to find a way to facilitate better conversations in the edk2 community, we decided we would try using "GitHub Discussions".  Personally, I don't love all things GitHub, but I have found them to be adequate and they have a lot of benefits in that they are easy to use, well documented, well understood, well supported, always being "improved", and run for free without more overhead (accounts, services, servers, security patches, etc).  By going this route this community can focus on firmware related work rather than web services. 

I'll be opening a number of new discussions about code layout/repos, tooling, CI, edk2 development process, etc.  Hopefully you all feel empowered to share your thoughts and bring up the topics that matter to you and your business/projects.  Maybe with a few good discussions there can be better alignment and understanding across the broad edk2 project and UEFI firmware community.

With that, I'll leave this discussion thread for others to chime in with any relevant thoughts.  

Thanks
Sean

**Comment by CalculonPrime** [comment]
I have had great trouble trying to join and post to the edk2 groups to get help.  Once joined, all messages I post report they are being held for moderation, but they are never released, and I am unable to contact the moderator or get support.  I then tried to post a message to the help group asking for help, but encountered the same issue - posts moderated.

So, I was happy to see this thread posted.  Perhaps I can actually receive help through this means.
